File: StepByStep/정수론및조합론/sokim/5086.py
import sys

while True:
	a, b = map(int, sys.stdin.readline().rstrip().split())
	if a == 0 and b == 0:
		break
	if a % b == 0:
		print("multiple")
	elif b % a == 0:
		print("factor")
	else:
		print("neither")

File: boj/wmo/5086.c
#include <stdio.h>

int main() {

    int n, m;
    n = 1;
    m = 1;
    while (n != 0 || m != 0) {
        scanf("%d %d", &n, &m);
        if (n == 0 && m == 0)
            return 0;
    if (n == 0 || m == 0)
        printf("neither\n");
	if (!(n % m))
	    printf("multiple\n");
	else if (!(m % n))
	    printf("factor\n");
	else
	    printf("neither\n");
    }
    return 0;
}
